modules/devguide/examples/c/bulk-get.cc

```diff
@@ -70,6 +70,7 @@ main()
 
     // Make a list of keys to store initially
     std::vector<std::string> keys_to_get{ "foo", "bar", "baz" };
+    // tag::batching
```

Comment:
whitespace: better before the tag than after?

Reply:
I used `clang-format` and I think it aligned it this way because it treats it as regular comment.